### DISCUSSION TITLE
Asset manager creating false/default image icon when it is dropped to canvas area

### DISCUSSION THREAD

**Discussion by ChanduRach** [GrapesJS/grapesjs#4810] [Q&A]
When the asset manager is dropped to the canvas area it opens up the modal and when we close the modal without selecting or uploading any images it's leaving the default image on the canvas area.

how can I remove or how can I make manager to stop appending default image?

![image](https://user-images.githubusercontent.com/25901390/209667151-b0bdcb56-640d-40f1-884f-d2e2865076ee.png)

**Comment by artf** [comment]
You can update the image component with your [desired default images](https://github.com/artf/grapesjs/blob/7967881d217ec7510a8a75f954e81b73c86fedb0/src/dom_components/model/ComponentImage.js#L21-L29).
